### DISCUSSION TITLE
You're famous @Jyers

### DISCUSSION THREAD

**Discussion by tiagofreire-pt** [Jyers/Marlin#751] [General Discussion]
Man, you're doing an huge impact on Ender 3 V2 community. So that, many famous youtubers are promoting this firmware with good reviews!

Seems to even manufaturers are forking your repo!

ModBot
https://www.youtube.com/watch?v=EY87ai3-IXo

CHEP:
https://www.youtube.com/watch?v=TroPjdkSEOY

CHEP @ Voxlab:
https://www.youtube.com/watch?v=bgY8vwv3gQA

EB3D Printing
https://www.youtube.com/watch?v=ZqFIpoekTXY

**Comment by dkustec** [comment]
JyersUI FTW!
